.env.database

```diff
@@ -0,0 +1,15 @@
+# For local database
+DB_DATABASE=
+DB_PORT=
+DB_USER=
+DB_PASSWORD=
+DB_ROOT_PASSWORD=
+DB_HOST=
+
+# For docker database (Don't change the HOST=mysql because this is the docker service)
+DOCKER_DB_DATABASE=
+DOCKER_DB_PORT=
+DOCKER_DB_USER=
+DOCKER_DB_PASSWORD=
+DOCKER_DB_ROOT_PASSWORD=
+DOCKER_DB_HOST=mysql
```

Comment:
should we move this to .env.example ?

Reply:
Yes, I don't move because other people edit this file